huffup/huffup

```diff
@@ -120,8 +120,8 @@ main() {
 
     # Compute the URL of the release tarball in the Huff repository.
     RELEASE_URL="https://github.com/${HUFFUP_REPO}/releases/download/${HUFFUP_TAG}/"
-    BIN_TARBALL_URL="${RELEASE_URL}huffc${HUFFUP_VERSION}_${PLATFORM}_${ARCHITECTURE}.tar.gz"
-    MAN_TARBALL_URL="${RELEASE_URL}huffc${HUFFUP_VERSION}.tar.gz"
+    BIN_TARBALL_URL="${RELEASE_URL}huff_${HUFFUP_VERSION}_${PLATFORM}_${ARCHITECTURE}.tar.gz"
+    MAN_TARBALL_URL="${RELEASE_URL}huff_man_${HUFFUP_VERSION}.tar.gz"
```

Comment:
y this?

Reply:
`stage` isn't synced up with `main` rn- merged `main` into this PR

https://github.com/huff-language/huff-rs/commit/a5a95868f89bc593ed38cbbc06e8913ecdac59a5